SplitFiles/base.c

```diff
@@ -17,7 +17,7 @@ GameParameters* get_game_parameters() {
     params->breadBin = NULL;
     params->reputation = 1000; //Start with 1000 rep
     params->hotelOpen = 1;     //Open by default
-    params->maxCustomers = 6;  //Start with a reasonable cuustomeer queue, goes up each day
+    params->maxCustomers = 4;  //Start with a reasonable cuustomeer queue, goes up at start of each day
```

Comment:
This actually results in a starting count of 5 now - was always 6 customer max before